src/lib/components/Sort/Sort.js

```diff
@@ -93,7 +93,10 @@ Sort.propTypes = {
 Sort.defaultProps = {
   currentSortBy: null,
   currentSortOrder: null,
-  label: (cmp) => cmp,
+  label: (cmp) => <>
+      <label className="min-width-max-content rel-mr-1">Sort by</label>
+      {cmp}
```

Comment:
if someone provides their own label, the displayed component will end up with 2 labels